NASA-IMPACT/veda-ui
# Floating Data Catalog Filters Sidebar Follow-up and starting some cleaning and modularizing

**Related Ticket:**

Closes https://github.com/NASA-IMPACT/veda-ui/issues/926

### Description of Changes
* Make filters sidebar height `100%` when filters sidebar height (when all expanded) is greater than or equaled to the height of all the cards
* Calculate filters sidebar height with `position:sticky` when filters sidebar height (when all expanded) is less than height all of cards
* Display all filters opened now
* Move `prepareDatasets` to own logic file - cleaning
* Move layout components from Data Catalog component into the wrapper - starting to modularize

### Notes & Questions About Changes
**DEMO Here:**
https://www.loom.com/share/5f88edba7ca4422d9f2636464cebf7b6?sid=62ae3536-4174-4de0-adfe-a19b2e4a5893

### Validation / Testing
* Make sure when the height of the filters sidebar (when expanded) is greater than the height of all the cards, the sidebar doesn't behave sticky
* Make sure when the height of the filters sidebar (when expanded) is less than the height of all the cards, the sidebar behaves sticky
